src/pharmpy/tools/resmod/tool.py

```diff
@@ -121,6 +126,33 @@ def _create_power_model(input_model):
     return model
 
 
+def _create_combined_model(input_model):
+    base_model = input_model
+    model = base_model.copy()
+    remove_error_model(model)
+    stats, y, f = _preparations(model)
```

Comment:
_preparations is a private function so it shouldn't be used here. Given my previous comment it shouldn't be needed.

Reply:
Yes, thank you! Now I understand more about the statements:)